src-java/grpc-speaker/grpc-service/build.gradle

```diff
@@ -20,7 +22,7 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-log4j2")
     implementation("org.springframework.boot:spring-boot-starter-web")
     implementation("org.springframework.security:spring-security-config:6.2.1")
-    implementation("org.springframework.security:spring-security-web")
+    implementation("org.springframework.security:spring-security-web:6.2.1")
```

Comment:
I believe the version is selected based on the spring-boot-dependencies version on line 16. Maybe it's better to update it? Once spring-boot-dependencies is 3.2.2, security-web is selected to be 6.2.1